packages/create-expo-app/src/utils/npm.ts

```diff
@@ -244,32 +234,49 @@ function isNpmPackageInfo(item: any): item is NpmPackageInfo {
   );
 }
 
+async function fileExistsAsync(path: string): Promise<boolean> {
+  try {
+    const stat = await fs.promises.stat(path);
+    return stat.isFile();
+  } catch {
+    return false;
+  }
+}
+
 export async function downloadAndExtractNpmModuleAsync(
   npmName: string,
   props: ExtractProps
 ): Promise<void> {
-  const cachePath = getCacheFilePath();
+  const cachePath = getTemporaryCacheFilePath();
 
   debug(`Looking for NPM tarball (id: ${npmName}, cache: ${cachePath})`);
 
   await fs.promises.mkdir(cachePath, { recursive: true });
 
-  const info = await getNpmInfoAsync(npmName);
+  const info = await getNpmInfoAsync(npmName, cachePath);
 
+  const cacheFilename = path.join(cachePath, info.filename);
   try {
-    const cacheFilename = path.join(cachePath, info.filename);
+    // TODO: This cache does not expire.
```

Comment:
Is this TODO important to fix now?

Reply:
possibly, if we want remote templates to eventually expire without the computer restarting.

Reply:
Restarting the computer doesn't clear the cache (on Windows).